### PR TITLE
chcon: move help strings to a markdown file

### PR DESCRIPTION
**Pull request by zleyyij**
#4368 
In this change, the `ABOUT` and `USAGE` help strings were moved to `chcon.md`.

### CODE REVIEW

File: src/uu/chcon/chcon.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# chcon
+
+```
+chcon [OPTION]... CONTEXT FILE... \n    \
+chcon [OPTION]... [-u USER] [-r ROLE] [-l RANGE] [-t TYPE] FILE... \n    \
+chcon [OPTION]... --reference=RFILE FILE...
```
Comment (tertsdiepraam) [inline src/uu/chcon/chcon.md:6]: This file needs a spell-checker:ignore line, but it looks good!

**Comment by tertsdiepraam** [comment]
Just a tip so you don't have to wait for the CI each time. You can run the spell checker yourself either via VS Code with [this extension](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker) or by running [`cspell`](https://cspell.org/) in the terminal.

**Comment by zleyyij** [comment]
Understood, thanks for the tip :)
That was resolved

File: src/uu/chcon/chcon.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<!-- spell-checker:ignore (vars) RFILE -->
+# chcon
+
+```
+chcon [OPTION]... CONTEXT FILE... \n    \
```
Comment (tertsdiepraam) [inline src/uu/chcon/chcon.md:5]: The `\n    \` is not supposed to be there. (Also on the next line)

File: src/uu/chcon/src/chcon.rs [hunk 3]
```diff
@@ -22,12 +22,8 @@ mod fts;
 use errors::*;
 
 static VERSION: &str = env!("CARGO_PKG_VERSION");
-static ABOUT: &str = "Change the SELinux security context of each FILE to CONTEXT. \n\
-                      With --reference, change the security context of each FILE to that of RFILE.";
-const USAGE: &str = "\
-    {} [OPTION]... CONTEXT FILE... \n    \
-    {} [OPTION]... [-u USER] [-r ROLE] [-l RANGE] [-t TYPE] FILE... \n    \
-    {} [OPTION]... --reference=RFILE FILE...";
+static ABOUT: &str = help_about!("chcon.md");
```
Comment (tertsdiepraam) [inline src/uu/chcon/src/chcon.rs:25]: Let's make this all `const` for consistency.

File: src/uu/chcon/src/chcon.rs [hunk 4]
```diff
@@ -1,11 +1,11 @@
 // spell-checker:ignore (vars) RFILE
-
 #![allow(clippy::upper_case_acronyms)]
 
 use clap::builder::ValueParser;
 use uucore::error::{UResult, USimpleError, UUsageError};
 use uucore::format_usage;
 use uucore::{display::Quotable, show_error, show_warning};
+use uucore::{help_about, help_usage};
```
Comment (tertsdiepraam) [inline src/uu/chcon/src/chcon.rs:8]: Maybe we can merge these?

**Comment by tertsdiepraam** [comment]
All good now! For next time you can squash or amend your commits to keep the history a bit cleaner. I'll squash them on merge now. Thanks!

**Pull request merged**